content/cloud-docs/api-docs/providers.mdx

```diff
@@ -254,79 +312,411 @@ curl \
 ```
 
-### Sample Response (public provider)
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "provver-y5KZUsSBRLV9zCtL",
+      "type": "registry-provider-versions",
+      "attributes": {
+        "version": "3.1.1",
+        "created-at": "2022-02-11T19:16:59.876Z",
+        "updated-at": "2022-02-11T19:16:59.876Z",
+        "key-id": "32966F3FB5AC1129",
+        "protocols": ["4.0"],
+        "permissions": {
+          "can-delete": true,
+          "can-upload-asset": true
+        },
+        "shasums-uploaded": true,
+        "shasums-sig-uploaded": true
+      },
+      "relationships": {
+        "registry-provider": {
+          "data": {
+            "id": "prov-cmEmLstBfjNNA9F3",
+            "type": "registry-providers"
+          }
+        },
+        "platforms": {
+          "data": [
+            {
+              "id": "provpltfrm-GSHhNzptr9s3WoLD",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-A1PHitiM2KkKpVoM",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-BLJWvWyJ2QMs525k",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-qQYosUguetYtXGzJ",
+              "type": "registry-provider-platforms"
+            },
+            {
+              "id": "provpltfrm-pjDHFN46y193bS7t",
+              "type": "registry-provider-platforms"
+            }
+          ],
+          "links": {
+            "related": "/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions/3.1.1/platforms"
+          }
+        }
+      },
+      "links": {
+        "shasums-download": "https://archivist.terraform.io/v1/object/dmF1b...",
+        "shasums-sig-download": "https://archivist.terraform.io/v1/object/dmF1b..."
+      }
+    }
+  ],
+  "links": {
+    "self": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "first": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20",
+    "prev": null,
+    "next": null,
+    "last": "https://app.terraform.io/api/v2/organizations/hashicorp/registry-providers/private/hashicorp/aws/versions?page%5Bnumber%5D=1&page%5Bsize%5D=20"
+  },
+  "meta": {
+    "pagination": {
+      "current-page": 1,
+      "page-size": 20,
+      "prev-page": null,
+      "next-page": null,
+      "total-pages": 1,
+      "total-count": 1
+    }
+  }
+}
+```
+### Note
+
+`shasums-uploaded` and `shasums-sig-uploaded` may be false if those files haven't been uploaded to Archivist yet. In this case, instead of including links to `shasums-download` and `shasums-sig-download`, the response will include upload links (`shasums-upload` and `shasums-sig-upload`) instead.
```

Comment:
may be --> will be

Reply:
the last instead at the end of the sentence is repetitive of the one at the beginning and can be removed

Reply:
also this is. a great thing to call out here 👍 👍